paddle/fluid/framework/ir/node.h

```diff
@@ -65,6 +65,8 @@ class Node {
 
   std::string Name() const { return name_; }
 
+  void SetName(const std::string name) { name_ = name; }
```

Comment:
Please create a new node with new name and replace this Node. If you update here, the underlying var_desc_ and op_desc_'s name will be out of sync.